index.js

```diff
@@ -309,3 +354,6 @@ module.exports.stringify = (data, {crlf = defaults.stringify.crlf, sections = de
 
   return `${output.trim()}${newline}`;
 };
+
+module.exports.parse = parse;
+module.exports.stringify = stringify;
```

Comment:
Are these necessary? Would prefer `const parse = module.exports.parse = () => {}` instead.

Reply:
This is a limitation of typescript jsdoc parsing. Anyway, removed them all and make d.ts something like written manually.